src/containers/layouts/articles/article3/article3.component.tsx

```diff
@@ -48,38 +55,29 @@ class Article3Component extends React.Component<Article3Props> {
     this.props.onCommentSubmit();
   };
 
-  private renderComment = (comment: string, key: number): React.ReactElement<CommentProps> => {
-    const { article } = this.props;
-
+  private renderComment = (comment: Comment, key: number): React.ReactElement<CommentProps> => {
     return (
-      <Comment
+      <CommentComponent
         key={key}
-        author={article.author}
         comment={comment}
-        date={article.date}
-        commentsCount={45}
-        likes={22}
-        onLikePress={() => {
-        }}
-        onCommentPress={() => {
-        }}
-        onProfilePress={() => {
-        }}/>
+        onLikePress={() => this.onLikeButtonPress(key)}
```

Comment:
is there any way to pass index into `CommentComponent` to avoid lambda inline?

Reply:
Yes, it's done but not refactored in every place.

Reply:
Refactored.